.github/workflows/checker-framework.yml

```diff
@@ -12,14 +12,14 @@ jobs:
       fail-fast: false
       matrix:
         profile:
-          - checker-framework-nullness-optional-interning
```

Comment:
good step, but we need to change line #1 also
`name: Checker-Framework` to `name: Checker`

Reply:
put name to job to remove yml file print 
```
jobs:
  checker-framework:
    name: analyse
```

Reply:
I got it! May I know how changing the word 'analyse' to 'job' would solve this failure in CI status?

Reply:
May be somewhere in doc of GitHub.
But you can check all names that we have in CI list and analyse corresponding yml file and you will see this.